### DISCUSSION TITLE
posts/Cuda_%E7%BC%96%E7%A8%8B%E6%A8%A1%E5%9E%8B

### DISCUSSION THREAD

**Discussion by giscus** [fffzlfk/blog#2] [Q&A]
# posts/Cuda_%E7%BC%96%E7%A8%8B%E6%A8%A1%E5%9E%8B

http://localhost:3000/posts/Cuda_%E7%BC%96%E7%A8%8B%E6%A8%A1%E5%9E%8B

**Comment by fffzlfk** [comment]
### 这是个测试

```rust
#[test]
fn test() {
    assert_eq!(1, 1);
}
```
